src/views/ExportHistoryDropdown.tsx

```diff
@@ -210,6 +214,63 @@ export const ExportHistoryDropdown = (props: ExportHistoryDropdownProps) => {
     selectedLocale,
   ]);
 
+  const allVaultTransfers = useLoadedVaultAccountTransfers();
+  const exportVaultTransfers = useCallback(async () => {
```

Comment:
can this be defined outside of the component so you dont need useCallback? like just inject stringGetter and other variables you need

Reply:
oh this isnt the first of this pattern... fine i guess 😆